### ISSUE TITLE
`/eth/v1/validator/liveness` does not accept string-encoded validator-indices

### ISSUE THREAD

**Issue by guybrush** [sigp/lighthouse#5182]
`/eth/v1/validator/liveness` does not accept string-encoded validator-indices

```
curl -H 'accept: application/json' -H 'Content-Type: application/json' -X GET "http://$ETHNODE/eth/v1/node/version"
{"data":{"version":"Lighthouse/v4.6.0-rc.0-2e8e160/x86_64-linux"}}

curl -H 'accept: application/json' -H 'Content-Type: application/json' -X POST -d '[1]' "http://$ETHNODE/eth/v1/validator/liveness/235999"
{"data":[{"index":"1","is_live":true}]}

curl -H 'accept: application/json' -H 'Content-Type: application/json' -X POST -d '["1"]' "http://$ETHNODE/eth/v1/validator/liveness/235999"
{"code":400,"message":"BAD_REQUEST: body deserialize error: Request body deserialize error: invalid type: string \"1\", expected u64 at line 1 column 4","stacktraces":[]}
```

**Comment by eserilev** [comment]
opening a PR for this

**Comment by michaelsproul** [comment]
Will be fixed in v5.0.0 thanks to #5184
